css/style.css

```diff
@@ -174,6 +178,33 @@ html{
     transform:  rotate(45deg) translate(-5px,-2px);
     
 }
+.user-profile {
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;  
+  border : 1px solid var(--underline);
+  border-radius: 100%;
+  box-shadow: 0 0 12px #181717a4;  
```

Comment:
define the variable of this box-shadow property and apply it everywhere you're using that color.

Reply:
I worked on this in the second pr